packages/test-utils/src/wrapper.js

```diff
@@ -582,3 +594,15 @@ export default class Wrapper implements BaseWrapper {
     this.element.dispatchEvent(event)
   }
 }
+
+export const enableAutoDestroy = (hook: Function) => {
+  if (Wrapper._instances) {
+    throwError('enableAutoDestroy cannot be called more than once')
+  }
+
+  Wrapper._instances = []
```

Comment:
initializing this here feels a little weird. should instead `isAutoDestroyEnabled = true`?

Reply:
I agree 🤔...

Just a thought, but I wonder if we should put all (or at least some) of this logic and tracking of instances in a separate `auto_destroy` module and just expose a way to hook into the constructor? Something like...

```javascript
// wrapper.js
// ref: https://nodejs.org/docs/latest-v10.x/api/events.html#events_class_eventemitter
import EventEmitter from 'events';

// Maybe this bit could even go in a separate module...
export const hooks = new EventEmitter();

export default class Wrapper  implements BaseWrapper {
  // ...
  constructor() {
    // ...
    hooks.emit('create', this);
  }
}
```

Reply:
brought this up in https://github.com/vuejs/vue-test-utils/issues/1224#issuecomment-497658163